admin-dev/themes/new-theme/js/components/grid/extension/action/bulk/category/delete-categories-bulk-action-extension.js

```diff
@@ -54,17 +54,22 @@ export default class DeleteCategoriesBulkActionExtension {
         const $checkboxes = grid.getContainer().find('.js-bulk-action-checkbox');
```

Comment:
```suggestion
        const $checkboxes = grid.getContainer().find('.js-bulk-action-checkbox:checked');
```

Having this additional selector would allow you to keep the existing code, and prevent having to filter all your results.

Reply:
Good idea, that should work ineed. :+1:

Reply:
Done.